pkg/kf/commands/config/config.go

```diff
@@ -229,6 +230,18 @@ func GetServiceCatalogClient(p *KfParams) servicecatalogclient.Interface {
 	return cs
 }
 
+// GetDynamicClient gets a dynamic Kubernetes client
+func GetDynamicClient(p *KfParams) dynamic.Interface {
```

Comment:
Can you explain what a dynamic client is a bit and why we need it?

Reply:
added, now the comment is

```
// GetDynamicClient gets a dynamic Kubernetes client. Dynamic clients can work
// on any type of Kubernetes object, but only support the common fields.
//
// Dynamic clients can be used to get alternative representations of objects
// like tables, or traverse multiple types of object in a single pass e.g. to
// construct a tree based on OwnerReferences.
```